packages/schematics/angular/workspace/schema.json

```diff
@@ -31,6 +31,11 @@
       "type": "boolean",
       "default": false,
       "x-user-analytics": 14
+    },
+    "strict": {
+      "description": "Creates a workspace with stricter TypeScript compiler options.",
+      "type": "boolean",
+      "default": true
```

Comment:
We should probably set this to `false` and get the PR in. And move the decision on whether we want to turn in on for 9.0 based on data we collect for this, later.

Reply:
I think `"noImplicitAny": true,` should be there regardless of the setting.  Without it, a developer is essentially bypassing the overwhelming majority of the benefits of TypeScript.

Reply:
I guess just adding a new flag is not enough to get noticed by developers. It may be better to make a prompt like "Would you like to enable TypeScript strict type checking? (y/N)". And I agree on the `false` by default.

Reply:
I think for sure we should

1) Turn off by default, add prompt so that it's more discoverable by users 
2) We revisit this based on the feedback we get right before version 9 RC.